.github/dependabot.yml

```diff
@@ -0,0 +1,14 @@
+version: 2
+updates:
+  - package-ecosystem: gomod
```

Comment:
just curious if dependabot will also update vendor dir or someone needs to run it manually ? :thinking: